string2int/string2int/string2int.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+
+int* str2int(const char* string)
+{
+	//decleration of index representing the position in the string
+	int i = 0;
+	int Negative = 0;
+
+	if (string[i] == 45)
+	{
+		Negative = 1;
+	}
+
+	//finding the amount of characters in the string
+	while (string[i] != '\0')
+	{
+		i += 1;
```

Comment:
this can be done more cleanly with pointer arithmetic. If you need 'pInputString' to always point to the beginning of the string, you can copy the pointer first:
char* pEndOfString = pInputString;
// Advances pointer string to the end
while (*pEndOfString++ != '\0');